packages/core/src/compiler/Style.ts

```diff
@@ -152,6 +151,8 @@ import {
 } from "utils/Util";
 import { checkTypeConstructor, isDeclaredSubtype } from "./Domain";
 
```

Comment:
Just a minor style nitpick: blank lines introduce groupings of `import`s, so I'd remove this one to allow this new `import` to be sorted in with the existing ones:

```suggestion
```